Django 3 compatibility

jaknel opened the pull request:
Django 3 removes 'six' from django.utils.  urllib.parse can be
imported directly on python3.

bgroff commented:
It looks like your build failures are because lxml is not compatible with Python3.4, I am not sure how this issue can be fixed at the moment and does not appear to be related to your changes.

mingchen commented:
Please update `tox.ini` to include django3 into test.

jaknel commented:
I've modified the tox.ini file to accept for the Django 3.0 beta, and dropped support for Python 3.5.  Looks like changes the tox.ini itself do not get checked by the Travis CI build, so not sure how to verify I updated them properly.
